components/analytics/system_analytics/system_analytics.tsx

```diff
@@ -42,6 +49,7 @@ export default class SystemAnalytics extends React.PureComponent<Props> {
         if (this.props.isLicensed) {
             AdminActions.getAdvancedAnalytics();
         }
+        this.pluginStats = await getPluginStats() as Record<string, PluginAnalyticsRow>;
```

Comment:
1/5 I suggest we add this as a prop rather than importing this function here, though it seems there are other actions being used here that are not passed in.